toml.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"io/ioutil"
+
+	"github.com/BurntSushi/toml"
+)
+
+type TOMLFile struct {
+	path string
+}
+
+func NewTOMLFile(path string) *TOMLFile {
+	return &TOMLFile{
+		path: path,
+	}
+}
+
+func (t *TOMLFile) Load() (map[string]string, error) {
```

Comment:
This looks great! As stated in the other request, I'd suggest inheriting from `*JSONFile` to use the `flatten` function instead of splitting it out. Check out `YAMLFile` as a reference.